controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/JobController.java

```diff
@@ -301,6 +301,8 @@ public void collectGarbage() {
                                    deactivateTester(TesterId.of(id), type);
                                    curator.deleteRunData(id, type);
                                    logs.delete(id);
+                                   controller.applications().applicationStore().removeAll(id);
```

Comment:
This doesn't need to be inside the for, everything for the application is deleted dependent of the `JobType`?

Reply:
Thanks.